portfolio/src/main/webapp/index.html

```diff
@@ -5,12 +5,14 @@
     <title>sehhaj portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+  </head> 
```

Comment:
generally we try to avoid trailing whitespace, not sure what editor you're using but a lot of them have the ability to strip trailing whitespaces on save

Reply:
I'm not sure what I did and how that whitespace ended up in the file? I think I have deleted it now. Use the Cloud Shell Editor?

Reply:
Ok so my quick delve into the could shell editor suggests it probably is possible if you figure out how to configure the right tasks, but there's no nice button like there is on a lot of editors.
This might be a good thing to ask the kitkat chat as they're much more familiar with that dev environment, they should be able to tell you if there's any autoformat options